src/usr/local/www/status_captiveportal_voucher_rolls.php

```diff
@@ -88,7 +88,7 @@
 		</thead>
 		<tbody>
 <?php
-			$voucherlck = lock("vouche{$cpzone}r");
+			$voucherlck = lock("voucher{$cpzone}");
```

Comment:
This has been odd for a *long* time.

Reply:
Bug. Every other instance of the file is referenced correctly. Rename is fine.